.github/workflows/pull_request.yml

```diff
@@ -8,16 +8,16 @@ on:
     branches:
       - main
 
-env:
-  ARC_TABLES_PORT: 8000
-  ARC_DB_EXTERNAL: true
-
 jobs:
   node:
     uses: nasa-gcn/.github/.github/workflows/node.yml@main
     secrets:
       CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
   playwright:
+    env:
+      CI: true
```

Comment:
Are you sure that GitHub Actions doesn't already set this automatically? I think that GitLab does.

Reply:
Yea, I think it should, but it doesn't seem like it is getting recognized in the code